src/server/routes/readings.js

```diff
@@ -42,6 +42,26 @@ router.get('/line/:meter_ids', async (req, res) => {
 	}
 });
 
+
+/**
+ * GET group readings by meter id for line chart
+ * @param {array.<int>} meter_ids
+ * @param {TimeInterval} [timeInterval]
+ */
+router.get('/line/groups/:group_ids', async (req, res) => {
+	// We can't do .map(parseInt) here because map would give parseInt a radix value of the current array position.
+	const groupIDs = req.params.group_ids.split(',').map(s => parseInt(s));
```

Comment:
Strongly recommend using the new jsonschema package to sanitize input, and respond the proper status code to the client. See routes/group.js for more information.

Reply:
This seems like something for a separate issue - perhaps a checklist of routes that need to have validation applied?